fanuc_cr7ia_support/package.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>fanuc_cr7ia_support</name>
+  <version>0.1.0</version>
+  <description>
+    <p>
+      ROS-Industrial support for the Fanuc CR-7iA (and variants).
+    </p>
+    <p>
+      This package contains configuration data, 3D models and launch files
+      for Fanuc CR-7iA manipulators. This currently includes the /L model
+      only.
+    </p>
+    <p><b>Specifications</b>:</p>
+    <ul>
+      <li>CR-7iA/L - Normal Range</li>
+    </ul>
+    <p>
+      Joint limits and maximum joint velocities are based on the information in
+      the FANUC Robot CR-7iA Mechanical Unit Operator's Manual.
+      All urdfs are based on the default motion and
+      joint velocity limits, unless noted otherwise (ie: no support for high
+      speed joints, extended / limited motion ranges or other options).
+    </p>
+    <p>
+      Before using any of the configuration files and / or meshes included
+      in this package, be sure to check they are correct for the particular
+      robot model and configuration you intend to use them with.
+    </p>
+  </description>
+  <author>Dave Niewinski</author>
+  <maintainer email="dniewinski@clearpathrobotics.com">dniewinski@clearpathrobotics.com</maintainer>
+  <maintainer email="g.a.vanderhoorn@tudelft.nl">G.A. vd. Hoorn (TU Delft Robotics Institute)</maintainer>
```

Comment:
I've added myself as maintainer, as we do that for all ROS-I packages.

Are you actually planning to maintain this? I'd very much welcome that, but it is a committment.